rov/rov.py

```diff
@@ -62,10 +62,8 @@ def init_hw(self):
 
         self.controls = controller(self.motor_control)
 
-        #""" Disabled until hardware is done and sw is tested
-        # self.IMU = IMU()
-        # self.pressure = Pressure()
-        #"""
+        self.IMU = IMU()
```

Comment:
Change self.IMU to self.imu

To keep to the convention of lowercase instances.